com.unity.netcode.gameobjects/Tests/Runtime/BaseMultiInstanceTest.cs

```diff
@@ -53,6 +56,51 @@ public virtual IEnumerator Teardown()
             yield return new WaitUntil(() => Time.frameCount >= nextFrameNumber);
         }
 
+        /// <summary>
+        /// We want to exclude the TestRunner scene on the host-server side so it won't try to tell clients to
+        /// synchronize to this scene when they connect
+        /// </summary>
+        private bool VerifySceneIsValidForClientsToLoad(int sceneIndex, string sceneName, LoadSceneMode loadSceneMode)
+        {
+            // exclude test runner scene
+            if (sceneName.StartsWith(k_FirstPartOfTestRunnerSceneName))
+            {
+                return false;
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// This registers scene validation with the server to prevent the server from telling connecting
```

Comment:
I'm a bit confused by your comments.
a few lines above, you say "We want to exclude the TestRunner scene on the host-server side so it won't"
but here you say "This registers scene validation with the server to prevent the server from"
is the test scene included or excluded?

Reply:
I will change that comment to:
>         /// This registers scene validation callback for the server to prevent it from telling connecting
>         /// clients to synchronize (i.e. load) the test runner scene.  This will also register the test runner
>         /// scene and its handle for both client(s) and server-host.

But the phrase is correct... this is called for every NetworkManager instance.  If the NetworkManager instance happens to be the server's, then it registers the VerifySceneIsValidForClientsToLoad callback.  

When the server is putting together the synchronization SceneEvent for each client this validation callback will be invoked for every scene the server already has loaded and if the callback returns false then the server will not include that scene as a scene the clients need to "synchronize".   The issue, primarily, is that the temporary Test runner scene isn't in the Scenes in Build list initially and will cause issues if the clients try to load it.
Since the clients and server-host share the same scenes loaded, we also have to include the Test runner scene (since most integration tests using this class don't change the currently active scene) in order to avoid the issue where NetworkObjects are being synchronized to the TestRunner scene.  While this won't crash anything, it will generate a warning and then use the currently active scene.  This just removes that warning to prevent spam as well as any confusion why they are getting the warning in the event a developer is trying to debug an issue.

Reply:
If you want, we can huddle briefly tomorrow and I can explain it possibly better that way and you can ask me any additional questions.  It is a little bit confusing at first.